Cargo.toml

```diff
@@ -34,10 +34,11 @@ members = [
 	"pallets/collator-allowlist",
 	"pallets/crowdloan-claim",
 	"pallets/crowdloan-reward",
-	"pallets/tinlake-investor-pool",
 	"pallets/migration",
 	"pallets/bridge",
 	"pallets/nft",
+	"pallets/pools",
+	"pallets/loans",
```

Comment:
nitpick. but I like them as pallet-loan and pallet-pool as singulars.
With parity substrate core pallet naming, i think they do not have a clear standard but I see most of them still use singulars.